pom.xml

```diff
@@ -99,6 +99,8 @@
         <moneta.version>1.4.2</moneta.version>
         <owasp-java-html-sanitizer.version>20220608.1</owasp-java-html-sanitizer.version>
         <logstash.version>7.4</logstash.version>
+        <tomcat-embeded.version>11.0.0-M13</tomcat-embeded.version>
```

Comment:
⚠️ We do not want this for several reasons. First, let's stick with the spring boot versions. Second, we do not want milestones. Last, but not least, the whole PowerAuth stack is running on Tomcat 10.